en_us.lang

```diff
@@ -5,43 +5,43 @@ gui.xaero_delete=Delete
 gui.xaero_OK=OK
 gui.xaero_add_edit=Add/Edit
 gui.xaero_new_waypoint=New Waypoint
-gui.xaero_initial=initial
+gui.xaero_initial=initials
```

Comment:
`gui.xaero_initial`, `gui.xaero_yaw`, and `gui.xaero_waypoint_name` are all lowercase. Is this intentional, as they are placeholders on the New Waypoint screen, or they should be capitalized as well?

Reply:
OK, that's what I thought.